cmd/create_prs/create_prs.go

```diff
@@ -131,3 +139,17 @@ func run(c *cobra.Command, _ []string) {
 		logger.Warnf("turbolift create-prs completed with %s %s(%s, %s, %s)\n", colors.Red("errors"), colors.Normal(), colors.Green(doneCount, " OK"), colors.Yellow(skippedCount, " skipped"), colors.Red(errorCount, " errored"))
 	}
 }
+
+func prDescriptionUnchanged(dir *campaign.Campaign) bool {
+	originalPrTitle := fmt.Sprintf("TODO: Title of Pull Request (%s)", dir.Name)
+	originalPrBody := `TODO: This file will serve as both a README and the description of the PR. Describe the pull request using markdown in this file. Make it clear why the change is being made, and make suggestions for anything that the reviewer may need to do.
+
+By approving this PR, you are confirming that you have adequately and effectively reviewed this change.
+
+## How this change was made
+TODO: Describe the approach that was used to select repositories for this change
+TODO: Describe any shell commands, scripts, manual operations, etc, that were used to make changes
+
+<sub>This PR was generated using [turbolift](https://github.com/Skyscanner/turbolift).</sub>`
```

Comment:
I think this might be prone to desync with the template. Could this instead compare with the template itself? I'm guessing you don't do that because the template isn't exported from the init command, is that right? Perhaps you could move the template into a package that both commands have access to.